RecoLocalTracker/SiStripClusterizer/plugins/SiStripClusterizerConditionsGPUESProducer.cc

```diff
@@ -0,0 +1,62 @@
+/**\class SiStripClusterizerConditionsGPUESProducer
+ *
+ * Create a GPU cache object for fast access to conditions needed by the SiStrip clusterizer
+ *
+ * @see SiStripClusterizerConditions
+ */
+#include <memory>
+
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+#include "RecoLocalTracker/Records/interface/SiStripClusterizerConditionsGPURcd.h"
+
+#include "CalibFormats/SiStripObjects/interface/SiStripGain.h"
+#include "CalibFormats/SiStripObjects/interface/SiStripDetCabling.h"
+#include "CalibFormats/SiStripObjects/interface/SiStripClusterizerConditionsGPU.h"
+
+class SiStripClusterizerConditionsGPUESProducer : public edm::ESProducer {
```

Comment:
Maybe this could be implemented as a specialisation of [ConvertingESProducerWithDependenciesT](https://github.com/cms-sw/cmssw/blob/master/HeterogeneousCore/CUDACore/interface/ConvertingESProducerWithDependenciesT.h) ?
See for example how it is used in [RecoLocalCalo/HcalRecProducers/src/HcalESProducersGPUDefs.cc](https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/HcalRecProducers/src/HcalESProducersGPUDefs.cc)